src/app/Console/Commands/Install.php

```diff
@@ -26,45 +31,191 @@ class Install extends Command
+        $this->infoBlock('Installing Backpack CRUD:', 'Step 1');
 
-        $this->line(' Publishing configs, views, js and css files');
+        // Publish files
+        $this->progressBlock('Publishing configs, views, js and css files');
         $this->executeArtisanProcess('vendor:publish', [
-            '--provider' => 'Backpack\CRUD\BackpackServiceProvider',
+            '--provider' => BackpackServiceProvider::class,
             '--tag' => 'minimum',
         ]);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating users table (using Laravel's default migration)");
+        // Create users table
+        $this->progressBlock('Creating users table');
         $this->executeArtisanProcess('migrate', $this->option('no-interaction') ? ['--no-interaction' => true] : []);
+        $this->closeProgressBlock();
 
-        $this->line(" Creating App\Http\Middleware\CheckIfAdmin.php");
+        // Create CheckIfAdmin middleware
+        $this->progressBlock('Creating CheckIfAdmin middleware');
         $this->executeArtisanProcess('backpack:publish-middleware');
+        $this->closeProgressBlock();
+
+        // Install Backpack Generators
+        $this->progressBlock('Installing Backpack Generators');
+        $process = new Process(['composer', 'require', '--dev', 'backpack/generators']);
+        $process->setTimeout(300);
+        $process->run();
+        $this->closeProgressBlock();
+
+        // Create users
+        $this->createUsers();
+
+        // Addons
+        $this->installAddons();
+
+        // Done
+        $url = Str::of(config('app.url'))->finish('/')->append('admin/');
+        $this->infoBlock('Backpack installation complete.', 'done');
+        $this->note("Go to <fg=blue>$url</> to access your new admin panel.");
+        $this->note('You may need to run `php artisan serve` to serve your Laravel project.');
+        $this->newLine();
+    }
+
+    private function createUsers()
+    {
+        $userClass = config('backpack.base.user_model_fqn', 'App\Models\User');
+        $userModel = new $userClass();
+
+        // Count current users
+        $currentUsers = $userModel->count();
+
+        $this->newLine();
+        $this->infoBlock('Creating an admin:', 'Step 2');
+        $this->note('Quickly jump in your admin panel, using the email & password you choose here.');
+        $this->note('Currently there '.trans_choice("{0} are <fg=blue>no users</>|{1} is <fg=blue>1 user</>|[2,*] are <fg=blue>$currentUsers users</>", $currentUsers).' in the database.');
+
+        $total = 0;
+        while ($this->confirm(' Add '.($total ? 'another' : 'an').' admin user?')) {
+            $name = $this->ask(' Name');
+            $mail = $this->ask(" {$name}'s email");
+            $pass = $this->secret(" {$name}'s password");
+
+            try {
+                $userModel->insert([
+                    'name' => $name,
+                    'email' => $mail,
+                    'password' => bcrypt($pass),
```

Comment:
Let's take into consideration that user migth be using a different hashing algorithm, or have configured their algorithm. 

We should probably be using `Hash::make` here. https://laravel.com/docs/9.x/hashing

Reply:
Another nice catch 😅

Reply:
Also done 👌